devbuild.sh

```diff
@@ -117,6 +114,7 @@ VERBOSE=false
 
 # Turn off all apps to build and choose default later
 DEFAULT_BUILD=true
+BUILD_CONDA="off"
```

Comment:
That's easy enough. I can add a `conda_only` target that will turn off the default build of app components in addition to turning on the conda build. I'd prefer not to turn off the default build when `conda` is specified because that drastically changes the original behavior of `devbuild.sh`.